alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertError.kt

```diff
@@ -17,15 +17,29 @@ package com.amazon.opendistroforelasticsearch.alerting.alerts
 
 import com.amazon.opendistroforelasticsearch.alerting.elasticapi.instant
 import com.amazon.opendistroforelasticsearch.alerting.elasticapi.optionalTimeField
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.io.stream.Writeable
 import org.elasticsearch.common.xcontent.ToXContent
 import org.elasticsearch.common.xcontent.XContentBuilder
 import org.elasticsearch.common.xcontent.XContentParser
 import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
 import java.io.IOException
 import java.time.Instant
 
-data class AlertError(val timestamp: Instant, val message: String) : ToXContent {
+data class AlertError(val timestamp: Instant, val message: String) : Writeable, ToXContent {
 
+    @Throws(IOException::class)
+    constructor(sin: StreamInput): this(
+        sin.readInstant(),
+        sin.readString()
```

Comment:
Label these with the attribute names to keep the documentation consistent throughout the package and please do so everywhere else in this PR.

Reply:
Added comments